core/src/main/java/io/confluent/kafka/schemaregistry/metrics/MetricsContainer.java

```diff
@@ -223,8 +224,8 @@ private static MetricsContext getMetricsContext(SchemaRegistryConfig config,
     Map<String, Object> metadata =
             config.originalsWithPrefix(CommonClientConfigs.METRICS_CONTEXT_PREFIX);
 
-    metadata.put(RESOURCE_LABEL_CLUSTER_ID, kafkaClusterId);
-    metadata.put(RESOURCE_LABEL_GROUP_ID,
+    metadata.put(RESOURCE_LABEL_KAFKA_CLUSTER_ID, kafkaClusterId);
+    metadata.put(RESOURCE_LABEL_CLUSTER_ID,
```

Comment:
Can we still keep RESOURCE_LABEL_GROUP_ID in the context?

Reply:
Done !